### PR TITLE
fixes to test script to obtain reproducible results

### PR DESCRIPTION
**Pull request by maxkazmsft**

### CODE REVIEW

File: scripts/test_all.sh [hunk 1]
```diff
@@ -103,44 +126,51 @@ cd ../../dutchf3_patch/local
 export CUDA_VISIBLE_DEVICES=2
```
Comment (sharatsc) [inline scripts/test_all.sh]: Do we test if the test anchored to CUDA_VISIBLE_DEVICES=2 earlier is done by now? It appears the previous job may be running in the background when this test is being called.

Comment (maxkazmsft) [reply scripts/test_all.sh]: This assumes ND40 VM (or any VM with 8 GPUs) - all tests are run at the same time on different GPUs.

File: scripts/test_all.sh [hunk 2]
```diff
@@ -1,13 +1,25 @@
 #!/bin/bash
 
+# specify pretrained HRNet backbone
+PRETRAINED_HRNET='/home/maxkaz/models/hrnetv2_w48_imagenet_pretrained.pth'
```
[the remark this answers is not in the document]
Comment (maxkazmsft) [reply scripts/test_all.sh]: on DS VM there are different mounts and partitions across 3 filesystem mounts - do you still want to define ROOT? Data and models will be on different partitions, results will possibly go to temporary partition/mount

**Comment by maxkazmsft** [comment]
closing since other PR is a duplicate

**Pull request closed without merging**